### ISSUE TITLE
linter: is `import/import-no-namespace`'s `ignore` option working properly?

### ISSUE THREAD

**Issue by sebastienbarre** [oxc-project/oxc#15007]
### What version of Oxlint are you using?

latest

### What command did you run?

Through VSCode's oxc 1.24.0 extension

### What happened?

One my TypeScript file has the following import:
```typescript
import * as z from 'zod';
```
Another one has:
```typescript
import type * as DrizzleKit from 'drizzle-kit/api';
```

Both lines are getting flagged by the `import/import-no-namespace` rule. This is a rule I'd like to use, but as far as Zod is concerned, importing the whole namespace is the recommended way.

Now instead of littering my code with `// oxlint-disable-line import/no-namespace` each time I use Zod (which seems to trigger Copilot too), I figured I would add an exception for that rule straight into my `.oxlintrc.json`.

[The documentation](https://oxc.rs/docs/guide/usage/linter/rules/import/import-no-namespace.html#options) confuses me though (besides `ignore` being spelled `ignores` in the example):

`ignore`: array of glob strings for modules that should be ignored by the rule :
```json
{
  "ignore": ["*.json"]
}
```

I can't read Rust for the life of me, so I'm not sure what's going on with the `ignore` parameter, but this:
```
"import/no-namespace": ["warn", { "ignore": ["zod"] }],
```
will disable the rule for BOTH `'zod'` and `'drizzle-kit/api'`, whereas it should only disable it for `'zod'` and still warn me about the second one.
 
What kind of globbing is really performed here?

Thank you

**Comment by connorshea** [comment]
Actually, there's something weird about this doc page you've linked. The rule hasn't been called import/import-no-namespace for a while. Are we intentionally including older versions of pages here or is this an odd issue with the website generation logic? And how did you get to that doc page? It isn't listed in the rules list in the docs.

**Comment by sebastienbarre** [comment]
Hmm. I assume Google took me there. Is it deprecated?

**Comment by sebastienbarre** [comment]
Thanks, I'll test the fix!

If that helps, `import/import-no-namespace` is indeed mentioned in the Oxc Linter 's search results, in your documentation, *before* `import/no-namespace`:

<img width="919" height="232" alt="Image" src="https://github.com/user-attachments/assets/081df77f-a53f-4cf2-8e6b-c82489e85fb5" />

**Comment by connorshea** [comment]
That particular page only exists because it wasn't possible to have the same name for rules in different namespaces (e.g. typescript/no-namespace and import/no-namespace). But that was fixed so it got renamed, I don't really know why the old `import/import-no-namespace` page still exists.
